CountMatrixCloneDetection/VariableName.cs

```diff
@@ -94,7 +100,10 @@ public int[] ToArray()
                 ThirdLevelMemberAccessed,
                 SecondLevelMemberAccessed,
                 FirstLevelMemberAccessed,
-                DefinedByType
```

Comment:
I was thinking to disable this feaure (DefinedByType).  According to paper below  int i =0 and double i= 0.0 are considered duplicates.  Also the hash functions is making scale large and it is hard to tell the truth difference.

http://research.cs.queensu.ca/home/cordy/Papers/RCK_SCP_Clones.pdf

Reply:
Oh I merged it too fast then :P
Regarding the hash function, isn't it good that the difference is huge because they truly are different classes and hence should not be classified as clone.

Similarily, for the int vs double, I clubbed them together to 1, 2 and 3 and clubbed string and char as 4, 5.
So there would be minute differences showing up, which I believe should be there

Reply:
We can fine tune these features once we have finalized the test codebase

Reply:
If you group them it should be fine. For hash my comment stands. Since we already have features which handle object member access and I think this will just take our measurement out of scale.
Count is usually small number, hash will not be in that range